### DISCUSSION TITLE
Import Stack 1 WS not working.

### DISCUSSION THREAD

**Discussion by DanyRamdas** [stacks-task-manager/stacks2#6] [Bugs]
I follow the steps but could not press import workspace when it is blue, nothing does happen.

**Comment by DanyRamdas** [comment]
Working after I added an demo space.

**Reply by synapse** [reply]
Could not replicate it. Could you give me a step-by-step guide on how to reproduce it?

**Comment by sarraro** [comment]
Same issue for me. I can't import from version 1 - 
It seems to me (but i'm not sure) that the value of the source folder is copied to the target folder when the first or second is selected. Is there a log file to send you?

**Reply by synapse** [reply]
This should be fixed in beta 9
